packages/recommend/src/methods/getTrendingItemsForFacet.ts

```diff
@@ -0,0 +1,17 @@
+import { BaseRecommendClient, TrendingItemsForFacetQuery, WithRecommendMethods } from '../types';
+import { getTrendingGlobalItems } from '.';
+
+type GetTrendingItemsForFacet = (
+  base: BaseRecommendClient
+) => WithRecommendMethods<BaseRecommendClient>['getTrendingItemsForFacet'];
+
+export const getTrendingItemsForFacet: GetTrendingItemsForFacet = base => {
+  return (queries: readonly TrendingItemsForFacetQuery[], requestOptions) => {
+    return getTrendingGlobalItems(base)(
+      queries.map(query => ({
+        ...query,
+      })),
```

Comment:
```suggestion
      queries,
```

this seems equivalent (you have a similar pattern in some other places)

Reply:
Good remark, I forget to add the model attributes